### PR TITLE
Fix clang warnings

### PR DESCRIPTION
**Pull request by uweseimet**

### CODE REVIEW

File: src/raspberrypi/devices/scsi_host_bridge.cpp [hunk 1]
```diff
@@ -1066,10 +1066,8 @@ void SCSIBR::FS_GetCapacity(BYTE *buf)
 	ASSERT(fs);
 	ASSERT(buf);
 
-	int i = 0;
```
Comment (rdmark) [inline src/raspberrypi/devices/scsi_host_bridge.cpp:1069]: @uweseimet At a glance, these changes seem to alter the logic of the host bridge code. Are you confident this won't break the host bridge?

Comment (uweseimet) [reply src/raspberrypi/devices/scsi_host_bridge.cpp:1069]: @rdmark According to the clang compiler warnings the value of i is never used anywhere. I verified this and agree with the compiler. When you look at the surrounding code (this is just a few lines) you can probably confirm.

Comment (rdmark) [reply src/raspberrypi/devices/scsi_host_bridge.cpp:1069]: Ah, you're absolutely right.

**Review by rdmark — APPROVED** [review]

File: src/raspberrypi/devices/scsi_host_bridge.cpp (hunk 1 above)
Comment (uweseimet) [reply src/raspberrypi/devices/scsi_host_bridge.cpp:1069]: @rd Let's say, the compiler was right ;-). I was surprised that gcc did not report this. This shows once more that using different compilers helps with identifying potential issues.

**Pull request merged**